pages/index.tsx

```diff
@@ -1,88 +1,92 @@
 import Head from "next/head";
 import Image from "next/image";
+import Hero from "./Hero";
 import Navbar from "./Navbar";
+import GetInvolved from "./GetInvolved";
 
 export default function Home() {
   return (
     <>
       <Navbar />
+      <Hero />
+      <GetInvolved />
```

Comment:
And remove this one as well please? It should render okay afterwards.

Reply:
removed both